src/Microsoft.AspNetCore.SignalR.Core/HubConnectionContext.cs

```diff
@@ -271,7 +271,7 @@ public virtual void Abort()
             Input.CancelPendingRead();
 
             // We fire and forget since this can trigger user code to run
-            Task.Factory.StartNew(_abortedCallback, this);
+            ThreadPool.QueueUserWorkItem(_abortedCallback, this);
```

Comment:
For my future reference, what's the reason for doing this over `Task.Factory.StartNew`?

Reply:
2 reasons:
- Use the global queue instead of the local queue (which means it's more "fair" for other scheduled work items). This ends up being faster most of the time.
- Don't need to allocate a task because this is fire and forget